test/unittest/hang.test.lua

```diff
@@ -0,0 +1,6 @@
+test_run = require('test_run').new()
+
+test_run:cmd('create server replica with rpl_master=default, script="unittest/replica-7f4d4895ff58.lua"')
+test_run:cmd('start server replica')
+test_run:cmd('cleanup server replica')
+test_run:cmd('delete server replica')
```

Comment:
Do you really think that it is obvious, why this code should fail?

Reply:
Fixed.